source/api/utilities/$.md

```diff
@@ -50,3 +50,31 @@ cy.wrap($li)
   .click()
   .should('have.class', 'active')
 ```
+
+# Notes
+
+## Cypress.$ vs. cy.$$
+
+You can also query DOM elements with `cy.$$`. But `Cypress.$` and `cy.$$` are different. 
+
+`Cypress.$` refers to `jQuery` function itself. You can do anything with `Cypress.$` if you can do it with `jQuery` function. So, the both of them below works fine.
+
+```js
+Cypress.$.each([1, 2, 3], (index, value) => {
+  expect(index).to.eq(value)
+}) // works
+```
+
+```js
+$.each([1, 2, 3], (index, value) => {
+  expect(index).to.eq(value)
+}) // also works
+```
+
+But `cy.$$` is a wrapper of `jQuery.fn.init` function. In other words, you can only query DOM elements with `cy.$$`. Because of that, the jQuery utility functions like `jQuery.each`, `jQuery.grep` don't work with `cy.$$`.
```

Comment:
```suggestion
But `cy.$$` is a wrapper of the `jQuery.fn.init` function. In other words, you can only query DOM elements with `cy.$$`. Because of that, the jQuery utility functions like `jQuery.each`, `jQuery.grep` don't work with `cy.$$`.
```